components/write/WritingPanelForm.tsx

```diff
@@ -45,14 +45,19 @@ function WritingPanelForm({
     submit();
   };
 
-  // set the values of form
+  // set the values of form formData (state) set with article draft
   useEffect(() => {
+    console.log("current form",getValues("thumbnail"));
     if (formData)
-      reset({
-        ...formData,
-        content: decodeURIComponent(formData.content),
-        thumbnail: undefined,
-      });
+      {
```

Comment:
Turnout setting `thumbnail` to undefined leading to it not being shown, everytime we update the `formData` state. Whoops

Reply:
Well, i could do it like : 

- Check if the current form thumbnail a.k.a `getValues("thumbnail")` is empty or not
- If YES > apply the `formData.thumbnail` to the form.
- If NO > reapply the current form `getValues("thumbnail")` to the reset function.

Reply:
I think that would be unnecessary-ish, i think i prefer your current approach, straight to the data source. So if we'd want to use it again somewhere, we won't need to do any manipulation again.

Reply:
Okay, gotcha.